test/integration/JobRoleIntegrationTest.ts

```diff
@@ -16,4 +16,18 @@ describe.skip('JobRoleIntegration', function () {
             return;
         }
     });
+
+    it('getJobRoleById should return job role details', async () => {
+        try {
+            const response: AxiosResponse = await axios.get(URL);  
+            
+            const data = response.data;
+            const status = response.status;
+
+            expect(status).to.equal(200);
+            expect(data).to.not.be.null;
+        } catch (e) {
+            return e.message;
+        }
+    })
```

Comment:
I find this try / catch logic confusing. I don't think this should be in this test. If an exception is thrown from our test, something is broken and the test will tell us. Catching it, the test won't tell us

Do we run these against the aws url yet?

Reply:
Right that makes sense I'll remove that. No we don't run them yet there are tickets for that in the backlog though.

Reply:
fixed!